lte/gateway/c/core/oai/tasks/nas5g/src/M5GPDUSessionEstablishmentRequest.cpp

```diff
@@ -115,8 +115,12 @@ int PDUSessionEstablishmentRequestMsg::DecodePDUSessionEstablishmentRequestMsg(
           decoded += decoded_result;
         }
         break;
-      case REQUEST_5GSM_CAPABILITY_TYPE:
       case REQUEST_MAXIMUM_NUMBER_OF_SUPPORTED_PACKET_FILTERS_TYPE:
+        type_len = sizeof(uint8_t);
```

Comment:
add/modify unit tests for encoding/decoding.

Reply:
Done